File: Logical_Operater/2.py
input_id = input("Request id.\n")
input_pwd = input("Request pws.\n")
real_id = "egoing"
real_pwd = "11"
if real_id == input_id:
    if real_pwd == input_pwd:
        print("Hello!")
    else:
        print("Wrong pwd")
else:
    print("Wrong id")

File: Logical_Operater/2.rb
puts("Request id")
input_id = gets.chomp()
puts("Request pwd")
input_pwd = gets.chomp()
real_id = "egoing"
real_pwd = "11"
if real_id == input_id
  if real_pwd == input_pwd
    puts("Hello!")
  else
    puts("Wrong pwd")
  end
else
  puts("Wrong id")
end
